test/deploy/rackhd_ha_install.py

```diff
@@ -0,0 +1,217 @@
+    def test02_install_dhcp_config(self):
+        # create DHCP config
+        for vmnum in range(1, numvms + 1):
+            fit_common.remote_shell('echo INTERFACES=' + ifslist[1] + ' > /etc/default/isc-dhcp-server', vmnum=vmnum)
+            dhcp_conf = open('dhcpd.conf', 'w')
+            dhcp_conf.write('ddns-update-style none;\n'
+                            'option domain-name "example.org";\n'
+                            'option domain-name-servers ns1.example.org, ns2.example.org;\n'
+                            'default-lease-time 600;\n'
+                            'max-lease-time 7200;\n'
+                            'log-facility local7;\n'
+                            'deny duplicates;\n'
+                            'ignore-client-uids true;\n'
+                            'subnet 172.31.128.0 netmask 255.255.252.0 {\n'
+                            '  range 172.31.128.100 172.31.131.254;\n'
+                            '  option vendor-class-identifier "PXEClient";\n'
+                            '}\n')
+            dhcp_conf.close()
+            # copy file to ORA
+            fit_common.scp_file_to_ora('dhcpd.conf', vmnum)
+            os.remove('dhcpd.conf')
+            self.assertEqual(fit_common.remote_shell('cp dhcpd.conf /etc/dhcp/', vmnum=vmnum)['exitcode'],
+                             0, "DHCP Config failure.")
+
+    def create_hosts(self):
+        hosts_conf = open('hosts-conf', 'w')
+        for vmnum in range(1, numvms + 1):
+            line = '172.31.128.{}\tnode{}\n'.format(vmnum, vmnum)
+            hosts_conf.write(line)
+        hosts_conf.close()
+
+    def create_node_list(self):
+        result = ""
+        sb_net = self.get_southbound_network()
+        if sb_net:
+            for vmnum in range(1, numvms + 1):
+                result += "   node {{\n      ring0_addr: {1}.{0}\n      nodeid: {0}\n   }}\n".format(vmnum, sb_net)
+        return result
+
+    def find_southbound(self, httpEndpoints):
+        for i in httpEndpoints:
+            if i["routers"] == "southbound-api-router":
+                return i
+        return None
+
+    def get_template_file(self, file_name):
+        template_folder = './config_templates'
+        env = Environment(loader=FileSystemLoader(template_folder))
+        return env.get_template(file_name)
+
+    def get_southbound_network(self):
+        endpoints = fit_common.fitrackhd()['httpEndpoints']
+        southbound = self.find_southbound(endpoints)
+        if southbound and 'address' in southbound:
+            address = southbound["address"]
+            addrsplit = address.split('.')
+            return("{}.{}.{}".format(addrsplit[0], addrsplit[1], addrsplit[2]))
+        return None
+
+    def create_corosync_config(self):
+        template = self.get_template_file("corosync_config")
+        rendered = template.render(node_list=self.create_node_list())
+        return rendered
+
+    def test03_install_hosts_config(self):
```

Comment:
do you want this to depend on prior test results?   if so, add @depends(after=xxxx)